### PR TITLE
Fix pseudo ds

### PR DESCRIPTION
**Pull request by sjswerdloff**
issue #1242

### CODE REVIEW

File: lib/pymedphys/tests/experimental/pseudonymisation/test_pseudonymisation.py [hunk 1]
```diff
@@ -81,6 +81,29 @@ def test_pseudonymise_convenience_api():
...
+def test_identifier_with_DS_vr():
+    # PatientWeight is of type DS
+    # Because the strategy is only applied when the identifier is found
+    # in the dataset, the defect only surfaced in that circumstance
+    replacement_strategy = pseudonymisation_api.pseudonymisation_dispatch
+    logging.info("Using pseudonymisation strategy")
+    identifying_keywords = ["PatientID", "PatientWeight"]
+    ds_input = pydicom.Dataset()
+    ds_input.PatientID = "ABC123"
+    ds_input.PatientWeight = "73.2"
```
Comment (SimonBiggs) [inline lib/pymedphys/tests/experimental/pseudonymisation/test_pseudonymisation.py:94]: Not for this PR, but it might be of significant value to find these sorts of edge cases by using the Hypothesis testing library.

**Review by SimonBiggs — APPROVED** [review]
LGTM :)

File: lib/pymedphys/tests/experimental/pseudonymisation/test_pseudonymisation.py (hunk 1 above)
Comment (sjswerdloff) [reply lib/pymedphys/tests/experimental/pseudonymisation/test_pseudonymisation.py:94]: I'll investigate using hypothesis for further testing of the pseudonymisation strategy.
